overview.md

```diff
@@ -131,10 +131,7 @@ After the debug adapter has been initialized, it is ready to accept requests for
 Two requests exist for this:
 - [**launch**](./specification#Requests_Launch) request: the debug adapter launches the program ("debuggee") in debug mode and then starts to communicate with it.
 Since the debug adapter is responsible for the debuggee, it should provide options for the end user to interact with the debuggee.

```

Comment:
> Since the debug adapter is responsible for the debuggee, it should provide options for the end user to interact with the debuggee.

I'm not sure what this means exactly